vendor.conf

```diff
@@ -2,14 +2,14 @@
 github.com/urfave/cli v1.20.0
 github.com/kr/pretty v0.1.0
 github.com/kr/text v0.1.0
-github.com/containers/image 2c0349c99af7d90694b3faa0e9bde404d407b145
 github.com/containers/buildah 810efa340ab43753034e2ed08ec290e4abab7e72
 github.com/vbauerster/mpb v3.3.4
 github.com/mattn/go-isatty v0.0.4
 github.com/VividCortex/ewma v1.1.1
 golang.org/x/sync 42b317875d0fa942474b76e1b46a6060d720ae6e
 github.com/opencontainers/go-digest c9281466c8b2f606084ac71339773efd177436e7
-github.com/containers/storage v1.12.3
+github.com/containers/storage zstd https://github.com/giuseppe/storage
+github.com/containers/image zstd https://github.com/giuseppe/image
```

Comment:
(I think originally the idea was to list c/image, and then all it's dependencies. OTOH `go mod` is going to disrupt that anyway.)